src/app/groups/containers/group-log-view/group-log-view.component.html

```diff
@@ -55,12 +55,19 @@
                 <ng-template #iconSection>
                   <i class="ph-duotone item-icon" [ngClass]="{
                     'ph-flag-checkered': rowData.activityType === 'result_started',
-                    'ph-floppy-disk-back': rowData.activityType === 'saved_answer'
+                    'ph-floppy-disk-back': rowData.activityType === 'saved_answer' || rowData.activityType === 'current_answer'
                   }"></i>
                 </ng-template>
               </div>
               <div class="item-title">
                 <p class="item-title-caption">{{ rowData.activityType | logActionDisplay }}</p>
+                <a
+                  class="alg-link"
+                  [routerLink]="rowData.item | rawItemRoute | withAnswer: { id: rowData.answerId, loadAsCurrent: true } | url"
```

Comment:
Why "load as current"?   Is it like this somewhere else?
It looks like you could click on it, open it read-only and see afterwards if you want to load it as current.

BTW, is the current-user always allowed to view answers ? Doesn't it depends on his perms?

Reply:
Fixed